lib/Partitioner/Partitioner.cpp

```diff
@@ -1187,6 +1220,11 @@ llvm::Error Partitioner::Partition(CompilationContext &cctx) {
   // algorithm.
   F_ = selectRepFunc(module_, memSize_);
 
+  if (cctx.precisionConfig.quantMode == QuantizationMode::Profile) {
+    // Jump into profiling flow.
```

Comment:
You probably want the discussed comment about the early exit from the partitioner here.